Makefile

```diff
@@ -151,3 +145,81 @@ docs:  ## Build documentation
 .PHONY: docs-auto
 docs-auto:  ## Build and host docs with sphinx-autobuild
 	uv run --extra docs sphinx-autobuild -b html --open-browser --port=8080 --watch $(PROJECT) -W docs docs/_build
+
+---------------- : ## ----------------
+
+.PHONY: package
+package: package-pip  ## Build all packages
+
+.PHONY: package-pip
+package-pip:  ##- Build packages for pip (sdist, wheel)
+	uv run pyproject-build --installer uv .
+
+.PHONY: package-snap
+package-snap: snap/snapcraft.yaml  ##- Build snap package
+ifeq ($(shell which snapcraft),)
+	sudo snap install --classic snapcraft
+endif
+	snapcraft pack
+
+.PHONY: publish
+publish: publish-pypi  ## Publish packages
+
+.PHONY: publish-pypi
+publish-pypi: clean package-pip  ##- Publish Python packages to pypi
+	uv tool run twine check dist/*
+	uv tool run twine upload dist/*
```

Comment:
I see there is a `uv publish` command but no equivalent to `twine check`. Should we at least also use uv to publish?